adam/experiment/log_experiment.py

```diff
@@ -170,6 +176,19 @@ def learner_factory_from_params(
         return lambda: ObjectPursuitLearner.from_parameters(
             params.namespace("pursuit"), graph_logger=graph_logger
         )
+    elif learner_type == "pursuit-gaze":
+        return lambda: IntegratedTemplateLearner(
+            object_learner=PursuitObjectLearnerNew(
+                learning_factor=0.05,
+                graph_match_confirmation_threshold=0.7,
+                lexicon_entry_threshold=0.7,
+                rng=rng,
+                smoothing_parameter=0.002,
+                ontology=GAILA_PHASE_1_ONTOLOGY,
```

Comment:
This should target `GAILA_PHASE_2_ONTOLOGY` to match the curriculum default. Because this is hardcoded perhaps we should remove the option to even select the phase 1 versions?

Reply:
Left the ontology selection for now for debugging, but I will make an issue to remove it at a later date